Apps/W1/HybridGP/app/src/Migration/Support/CheckBooks/GPCheckbookMigrator.codeunit.al

```diff
@@ -37,9 +37,13 @@ codeunit 40025 "GP Checkbook Migrator"
     procedure MoveTransactionsData(BankAccountNo: Code[20]; BankAccPostingGroupCode: Code[20]; CheckbookID: Text[15])
     var
         GPCheckbookTransactions: Record "GP Checkbook Transactions";
+        GPCompanyAdditionalSettings: Record "GP Company Additional Settings";
         PostingAccountNumber: Code[20];
         Amount: Decimal;
     begin
+        if GPCompanyAdditionalSettings.GetMigrateOnlyBankMaster() then
```

Comment:
I am thinking this should be moved to MoveCheckbookStagingData where MoveTransactionsData is being called. 

if not GPCompanyAdditionalSettings.GetMigrateOnlyBankMaster() then
     MoveTransactionsData();

this will then catch upcoming changes where a beginning balance is created too. Bug 805.